src/cascadia/TerminalApp/JsonUtils.h

```diff
@@ -300,6 +317,27 @@ namespace TerminalApp::JsonUtils
         }
     };
 
+#ifdef WINRT_Windows_UI_H
+    template<>
+    struct ConversionTrait<winrt::Windows::UI::Color>
+    {
+        winrt::Windows::UI::Color FromJson(const Json::Value& json) const
+        {
+            return static_cast<winrt::Windows::UI::Color>(ConversionTrait<til::color>{}.FromJson(json));
+        }
+
+        bool CanConvert(const Json::Value& json) const
+        {
+            return ConversionTrait<til::color>{}.CanConvert(json);
+        }
+
+        std::string TypeDescription() const
+        {
+            return "color (#rrggbb, #rgb)";

```

Comment:
prefer `ConversionTrait<til::color>{}.TypeDescription()` to reduce number of times we have the string `"color (#rrggbb, #rgb)"` in code